snippets/luhnCheck.md

```diff
@@ -0,0 +1,19 @@
+### luhnCheck
+
+Implementation of the [Luhn Algorithm](https://en.wikipedia.org/wiki/Luhn_algorithm) used to validate a variety of identification numbers, such as credit card numbers, IMEI numbers, National Provider Identifier numbers etc.
+Works with numbers and strings alike
+``` js 
+const luhnCheck = num => { 
+    let arr = (num+'').split('').reverse()
+    let lastDigit = arr.splice(0,1)
+    let sum = arr.reduce((acc,val,i) => i%2!==0 ? acc + parseInt(val) : acc + (parseInt(val) * 2)%9 || 9,0)
```

Comment:
Should be `(acc,val,i) => i%2!==0 ? acc + parseInt(val) : acc + ((parseInt(val) * 2)%9 || 9);`

Reply:
@Chalarangelo No it works fine now. I mistakingly   `(acc,val,i) => ` twice . I shuld have checked the code before commenting.